modal.js

```diff
@@ -0,0 +1,132 @@
+const myModal = document.getElementById('myModal');
+const seeProject = document.querySelectorAll('.see-details');
+
+const Projects = [
```

Comment:
- [ ]  Kindly make sure the pop-up window does not take more than the available viewport height of the screen, as you can see I'm unable to close the pop-up window, please adjust your pop-up according to the Figma template. 📈✏️

https://user-images.githubusercontent.com/95243996/202266966-ecb52de8-e53f-4f21-b6ce-f1010e5b22e9.mp4